tlsobs-api/handlers.go

```diff
@@ -636,6 +638,45 @@ func PreflightHandler(w http.ResponseWriter, r *http.Request) {
 	w.Write([]byte("preflighted"))
 }
 
+func heartbeatHandler(w http.ResponseWriter, r *http.Request) {
+	val := r.Context().Value(ctxDBKey)
+	if val == nil {
+		w.WriteHeader(http.StatusInternalServerError)
+		return
+	}
+	db, ok := val.(*pg.DB)
+	if !ok || db.Ping() != nil {
```

Comment:
`db.Ping()` doesn't actually talk to the database, so the preferred approach is to run a `SELECT 1`.
https://github.com/golang/go/issues/7619